integration-tests/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "prepare:env": "cd ../ && pnpm build:libraries && pnpm build:services",
-    "test:integration": "vitest ."
+    "test:integration": "vitest"
```

Comment:
removed this so i can run tests with filter locally. (`pnpm --filter integration-tests test:integration document-collection`)